Applets/UI/MainScreen/ContentView.swift

```diff
@@ -71,6 +74,29 @@ struct ContentView: View {
             }
         }
     }
+
+    func authenticate() {
+        let context = LAContext()
+        var error: NSError?
+
+        // check whether biometric authentication is possible
+        if context.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: &error) {
+            // it's possible, so go ahead and use it
+            let reason = "We need to verify your identity to display sensitive information."
+
+            context.evaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, localizedReason: reason) { success, authenticationError in
+                // authentication has now completed
+                if success {
+                    // authenticated successfully
+                    isShowingProfilePage.toggle()
+                } else {
+                    // there was a problem
+                }
+            }
+        } else {
+            // no biometrics
```

Comment:
What if user doesn't have biometrics? Maybe we should request a password here?

Reply:
Done